From: Dmitry Shachnev <mitya57@debian.org>
Date: Thu, 28 Jun 2018 00:53:59 +0300
Subject: Remove usage of Markdown options in docs/markdown.rst

Upstream code for Markdown requires CommonMark 0.5.6, which is not yet
in Debian.
---
 docs/markdown.rst | 35 -----------------------------------
 1 file changed, 35 deletions(-)

diff --git a/docs/markdown.rst b/docs/markdown.rst
index d02f713..d37fe30 100644
--- a/docs/markdown.rst
+++ b/docs/markdown.rst
@@ -25,32 +25,6 @@ As of version 0.2.0, markdown (rather than only reStructuredText) can be include
        block
        ```
 
-The above example renders as follows:
-
-.. argparse::
-    :filename: ../test/sample.py
-    :func: parser
-    :prog: sample
-    :markdown:
-
-    A random paragraph
-
-    Heading 1
-    =========
-
-    [I'm a link to google](http://www.google.com)
-
-    ## Sub heading
-
-    ```
-    This
-     is
-      a
-      fenced
-     code
-    block
-    ```
-
 The `CommonMark-py <https://github.com/rtfd/CommonMark-py>`__ is used internally to parse Markdown. Consequently, only Markdown supported by CommonMark-py will be rendered.
 
 You must explicitly use the `:markdown:` flag, otherwise all content inside directives will be parsed as reStructuredText.
@@ -93,12 +67,3 @@ To render this as MarkDown rather than reStructuredText, use the `markdownhelp`
         :func: blah
         :prog: sample
         :markdownhelp:
-
-This will then be rendered as:
-
-.. argparse::
-    :filename: ../test/sample2.py
-    :func: blah
-    :prog: sample
-    :markdownhelp:
-
